ethsigner/core/src/integration-test/java/tech/pegasys/ethsigner/jsonrpcproxy/model/jsonrpc/SendRawTransaction.java

```diff
@@ -49,16 +50,16 @@ public String request() {
 
   @SuppressWarnings("unchecked")
   public String request(final Request<?, EthSendTransaction> request, final long chainId) {
-    final List<Transaction> params = (List<Transaction>) request.getParams();
-    final Transaction transaction = params.get(0);
+    final List<JsonObject> params = (List<JsonObject>) request.getParams();
+    final JsonObject transaction = params.get(0);
     final RawTransaction rawTransaction =
         RawTransaction.createTransaction(
-            valueToBigDecimal(transaction.getNonce()),
-            valueToBigDecimal(transaction.getGasPrice()),
-            valueToBigDecimal(transaction.getGas()),
-            transaction.getTo(),
-            valueToBigDecimal(transaction.getValue()),
-            transaction.getData());
+            valueToBigDecimal(transaction.getString("nonce")),
+            valueToBigDecimal(transaction.getString("gasPrice")),
+            valueToBigDecimal(transaction.getString("gas")),
+            transaction.getString("to"),
+            valueToBigDecimal(transaction.getString("value")),
+            transaction.getString("data"));
```

Comment:
same comment as before, should we create FIELD_NONCE, FIELD_GASPRICE somewhere?

Reply:
done